packages/discovery-provider/src/gated_content/helpers.py

```diff
@@ -61,7 +62,7 @@ def does_user_support_artist(
     return True if result else False
 
 
-def has_user_purchased_content(
+def has_user_purchased_track(
```

Comment:
nit: but this seems a little misleading - they didn't purchase the _track_ per se in the album scenario